packages/combobox/src/constants.ts

```diff
@@ -0,0 +1,42 @@
+import { typeScales } from '@leafygreen-ui/tokens';
+import { ComboboxSize } from './Combobox.types';
+
+/**
+ * Width of the widest character (in px)
+ */
+export const maxCharWidth: Record<ComboboxSize, number> = {
+  [ComboboxSize.Default]: typeScales.body1.fontSize,
+  [ComboboxSize.Large]: typeScales.body2.fontSize,
+};
+
+/**
+ * Size of combobox x & y padding (in px)
+ */
+export const comboboxPadding: Record<ComboboxSize, { x: number; y: number }> = {
+  [ComboboxSize.Default]: { x: 7, y: 5 },
```

Comment:
Where is this being used? I don't love the separation of the values here from the styles themselves.

Reply:
I don't love it either, but we need to use these same values in ~7 different locations in `Combobox.styles.ts`

Reply:
Why don't those values live in `Combobox.styles.ts`?